src/main/java/duke/util/Storage.java

```diff
@@ -0,0 +1,109 @@
+package duke.util;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.Duke;
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * duke.util.Storage class that deals with loading tasks from the file and saving tasks in the file.
+ *
+ * @author Pearlynn
+ */
+
+public class Storage {
+    private final String pathname;
```

Comment:
Variable names should be in camelCase.

```suggestion
    private final String pathName;
```